scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua

```diff
@@ -8,9 +8,11 @@ require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/titles")
 require("scripts/globals/magic")
+require("scripts/globals/mobs")
 -----------------------------------
 
 function onMobInitialize(mob)
+    mob:setMobMod(tpz.mobMod.ADD_EFFECT, 1)
```

Comment:
Is this our preferred practice instead of adding it to the DB?

Reply:
usually yes. I dunno if its still on the table but there was this plan to phase out most of the mod tabling for monsters. Families were to migrate to mixins for theirs, spawns were to go in scripts, pool mods I wasn't clear on.